typescript/rekognition-lambda-s3-trigger/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "rekognition-lambda-s3-trigger",
+  "version": "1.0.0",
+  "bin": {
+    "rekognition-lambda-s3-trigger": "bin/rekognition-lambda-s3-trigger.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "@aws-cdk/assert": "*",
+    "@types/node": "10.17.27",
+    "aws-cdk": "*",
+    "ts-node": "^9.0.0",
+    "typescript": "~3.9.7"
+  },
+  "dependencies": {
+    "aws-cdk-lib": "^2.8.0",
+    "aws-sdk": "^2.1062.0",
+    "source-map-support": "^0.5.16"
+  }
```

```suggestion
  "dependencies": {
    "aws-cdk-lib": "*",
    "aws-sdk": "^2.1062.0",
    "constructs": "*",
    "source-map-support": "^0.5.16"
  }
```

It looks like you're missing a required dependency though. Would you mind adding `"constructs": "^10.0.0",` to your package.json dependencies?

Without this change you'll get the following build error:

```sh
node_modules/aws-cdk-lib/pipelines/lib/private/application-security-check.d.ts:4:27 - error TS2307: Cannot find module 'constructs' or its corresponding type declarations.

4 import { Construct } from 'constructs';
```

By the way you'll need to update the lockfile after accepting this change. Run `npm install` then commit the lockfile changes.